task.sql

```diff
@@ -1,11 +1,15 @@
--- Use our database
 USE ShopDB; 
 
--- Some data should be created outside the transaction (here)
-
--- Start the transaction 
 START TRANSACTION; 
 
--- And some data should be created inside the transaction 
+INSERT INTO Orders (CustomerID, Date) 
+VALUES (1, '2023-01-01');
```

Comment:
Ensure that the `CustomerID` and `Date` values being inserted into the `Orders` table are valid and exist in the database. This is crucial for maintaining data integrity.